core/base/Module.php

```diff
@@ -146,7 +146,24 @@ public function getUrlRules()
      *
      */
     public $useAppViewPath = false;
-    
+
+    /**
+     * @var array mapping from action ID to view configurations.
+     * Each name-value pair specifies the configuration of a specifed oder wildcard action to a single view folder.
+     * The first match
+     * For example,
+     *
+     * ```php
+     * [
+     *   'default/index' => '@app/views/mymodule/default',
+     *   'login/info' => '@app/views/mymodule/login',
+     *   'default/*' => '@app/views/mymodule/default',
+     *   '*' => '@app/views/mymodule',
+     * ]
+     * ```
+     */
+    public $viewMap = [];
```

Comment:
please add a since tag

Reply:
Yes the mapping with `'default/*' => '@app/views/mymodule/default'` works and is covered with a unit test (see `viewmap/*`): https://github.com/luyadev/luya/pull/1843/files#diff-f664c55d8c4356b2a48a47890a687cdd

I use the php function `fnmatch` to check for a ending wildcard: https://github.com/luyadev/luya/pull/1843/files#diff-0fff98519c554897d6d8bd3aa450d2b2R46